### DISCUSSION TITLE
API per il magazzino

### DISCUSSION THREAD

**Discussion by antoniograzioli** [fattureincloud/api#443] [Q&A] [answered]
Per un cliente mi serve esportare in Power BI la giacenza di magazzzino dei diversi prodotti, ma non mi pare di vedere nulla nelle API riguardo il magazzino? E' possibile in qualche modo estrarre questo dato?

**Comment by emajo** [comment, marked as answer]
Ciao @antoniograzioli non abbiamo API per quanto riguarda il magazzino, ma se l'unico dato necessario è la giacenza allora è sufficiente utilizzare le api dei [prodotti](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/products) (campo data.stock_current).

Grazie,
Emanuele.

**Reply by antoniograzioli** [reply]
ok, grazie, provo a verificare lo stock_current, dovrebbe bastare la giacenza attuale

**Reply by antoniograzioli** [reply]
perfetto, grazie, sembra funzionare! Lo "stock_current" immagino sia la giacenza a magazzino attuale, corretto?

**Reply by emajo** [reply]
Esatto!

**Reply by antoniograzioli** [reply]
OK, perfetto, ho verificato, la giacenza attuale esce corretta in Power BI usando i prodotti, con il fieldset =detailed

**Comment by marioquark** [comment]
Mi unisco alla richiesta: anche io per scopo di analisi in Power BI avrei bisogno di esportare la lista dei movimenti di magazzino per articolo, ma non trovo nessuna API a riguardo. 
Se provo a ricostruire i movimenti unendo i dati di fatture, ricevute, ecc i dati non sono corretti perchè mancano tutti i movimenti eseguiti manualmente sulla pagina di magazzino (correzioni, carichi ecc).
Ci sono soluzioni?
Grazie

**Reply by barcigabri** [reply]
Buongiorno,
Purtroppo non abbiamo API per ottenere i movimenti del magazzino e in questo momento non abbiamo modo di fornire una soluzione alla tua richiesta.
Buona giornata,
Gabriele
